include/wlr/render/gles2.h

```diff
@@ -24,4 +25,16 @@ struct wlr_texture *wlr_gles2_texture_from_wl_drm(struct wlr_egl *egl,
 struct wlr_texture *wlr_gles2_texture_from_dmabuf(struct wlr_egl *egl,
 	struct wlr_dmabuf_attributes *attribs);
 
+struct wlr_gles2_texture_attribs {
+	GLenum target;
+	GLuint tex;
+
+	bool inverted_y;
+	bool has_alpha;
+};
```

Comment:
This design stops us from ever supporting multi-planar formats (excluding the ones the driver does for us), because multiple OpenGL textures may be involved.

Reply:
I think it will be reasonable to break this API when we get support for multiplanar formats.